media.css

```diff
@@ -0,0 +1,1400 @@
+    margin-bottom: 35px;
+    min-height: 200px;
+  }
+
+  .stories-item__description-extended {
+    width: 100%;
+    display: grid;
+    grid-template-columns: 1.5fr 0.5fr 1fr;
+  }
+
+  .arrow {
+  	width: 70px;
+    display: flex;
+    align-items: center;
+  }
+
+  .stories-item__date {
+    font-size: 15px;
+    line-height: 14px;
+    font-weight: bold;
+    color: #d1d1d1;
+    opacity: 1;
+    text-align: right;
+  }
+
+  .gallery {
+    display: block;
+    height: 15%;
+    background: linear-gradient(2.6deg, #fff 62.5%, #000 63%);
+    margin-top: -5px;
+    width: 100%;
+    padding-bottom: 125px;
+  }
+
+  .gallery__title {
+    font-size: 30px;
+    line-height: 29px;
+    color: #fff;
+    opacity: 1;
+    text-align: right;
+    text-transform: uppercase;
+    font-weight: bold;
+    margin-bottom: 13px;
+    margin-right: 43px;
+  }
+
+  .gallery-content {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: 2;
+    margin-left: 43px;
+    margin-right: 43px;
+  }
+
+  .gallery-content__item {
+    display: block;
+    margin-bottom: -4px;
+  }
+
+  .gallery__photo {
+    width: 100%;
+  }
+
+  .footer {
+    width: 100%;
+  }
+
+  .footer__content {
+    max-width: 88%;
+    padding-top: 60px;
+  }
+
+  .footer__column-title {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .footer-column__nav-link {
+  	max-width: 155px;
+  }
+
+  .footer__button {
+    width: 143px;
+    height: 26px;
+    margin-top: 5px;
+    margin-bottom: 30px;
+  }
+
+  .social-icon {
+    width: 65%;
+  }
+
+  .copyright {
+    padding-top: 70px;
+    padding-bottom: 40px;
+  }
+
+  .hidden-768 {
+  	display: none;
+  }
+
+  .hidden-1024 {
+    display: block;
+  }
+}
+
+@media (min-width: 375px) and (max-width: 767px) {
+  .wrapper {
+    max-width: 767px;
+    margin: 0 auto;
+  }
+
+  .header {
+    grid-template-columns: 1fr 1fr;
+    width: 100%;
+    height: 22px;
+    padding-bottom: 26px;
+    margin-bottom: 0;
+  }
+
+  .header__menu,
+  .header__search,
+  .header__user-block,
+  .header__cart,
+  .header__language-block {
+    display: none;
+  }
+
+  .header__logo-image {
+    display: none;
+  }
+
+  .header__logo-image_adaptive {
+    width: 104px;
+    height: 22px;
+    display: flex;
+    align-items: center;
+    margin-left: 48px;
+  }
+
+  .header__menu-bar {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-bar {
+    display: block;
+    width: 41px;
+    height: 17px;
+    margin-right: 47px;
+  }
+
+  .top-content {
+    background-image: url(img/header-img-375.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 191px;
+    margin-bottom: 38px;
+  }
+
+  .top-content__block {
+    padding-top: 66px;
+    padding-left: 48px;
+  }
+
+  .top-content__h1 {
+    font-size: 20px;
+    line-height: 19px;
+    margin-bottom: 9px;
+  }
+
+  .arrow {
+  	width: 46px;
+  	height: auto;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)